### PR TITLE
Adding the homepage carousel component

### PR DESCRIPTION
**Pull request by Rayane0823**

### CODE REVIEW

**Review by Cyrildab — APPROVED** [review]

**Review by LucasMerino10 — APPROVED** [review]

**Review by LucasMerino10 — APPROVED** [review]

**Review by pagouillart — APPROVED** [review]

File: frontend/src/components/Carousel/Carousel.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  justify-content: flex-end;
+  align-items: flex-end;
+}
+.carousel-item {
+  height: 89.5vh;
+  min-width: 100vw;
+  object-fit: cover;
+}
+
+.alice-carousel__prev-btn {
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: 2 class with same style ?

File: frontend/src/components/Carousel/Carousel.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  display: none;
+}
+.alice-carousel__next-btn {
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss:19]: 2 class with same style ?

File: frontend/src/components/Carousel/Carousel.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  height: 1rem;
+  border-radius: 50%;
+  transition: background-color 0.3s ease;
+}
+
+.hometext-container {
+  position: absolute;
+  background-color: $white;
+  opacity: $opacity-high;
+  top: 70vh;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: bottom is better

File: frontend/src/components/Carousel/Carousel.scss [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  left: 20vw;
+  width: 80vw;
+  flex-direction: row-reverse;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Seriously ?

File: frontend/src/components/Carousel/Carousel.scss [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+}
+.hometext {
+  font-family: $fontBold;
+  margin-left: 0.625rem;
+  margin-right: 0.625rem;
+  text-align: center;
+  font-size: 14.5px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: No px with sont-size

File: frontend/src/components/Carousel/Carousel.scss [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  font-family: $fontBold;
+}
+
+.carousel-slide-visit_button {
+  all: unset;
+  font-family: $fontBold;
+  color: $white;
+  position: absolute;
+  padding: 0.563rem 1.375rem;
+  font-size: 15px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Again here

File: frontend/src/components/Carousel/Carousel.scss [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+}
+.carousel-first-slide-visit_container {
+  background-color: rgba($blueColor, $opacity-high);
+  top: 20vh;
+  left: 5vw;
+}
+
+.carousel-first-slide-visit_text {
+  font-size: 30px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: And here

File: frontend/src/components/Carousel/Carousel.scss [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.carousel-first-slide-visit_button {
+  top: 89%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Bottom is better

File: frontend/src/components/Carousel/Carousel.scss [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.carousel-second-slide-visit_container {
+  background-color: rgba($redColor, $opacity-high);
+  top: 20vh;
+  left: 35vw;
+}
+
+.carousel-second-slide-visit_text {
+  font-size: 28px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Always no px to font-size

File: frontend/src/components/Carousel/Carousel.scss [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.carousel-second-slide-visit_button {
+  top: 89%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: bottom is better again

File: frontend/src/components/Carousel/Carousel.scss [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.carousel-third-slide-visit_container {
+  background-color: rgba($purpleColor, $opacity-high);
+  top: 20vh;
+  left: 5vw;
+}
+
+.carousel-third-slide-visit_text {
+  font-size: 30px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Here again

File: frontend/src/components/Carousel/Carousel.scss [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.carousel-third-slide-visit_button {
+  top: 89%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.scss]: Again

File: frontend/src/components/Carousel/Carousel.jsx [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+          />
+          <img
+            src="./src/assets/paysage.png"
+            className="carousel-item"
+            alt="Aquarelle d'un paysage réunionnais"
+          />
+        </AliceCarousel>
+        {currentSlide === 0 && (
+          <article className="carousel-slide-visit_container carousel-first-slide-visit_container">
+            <h1 className="carousel-slide-visit_text carousel-first-slide-visit_text">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.jsx]: 3 h1 in same page ?

File: frontend/src/components/Carousel/Carousel.jsx [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+              type="button"
+              className="carousel-slide-visit_button carousel-first-slide-visit_button"
+            >
+              Visiter
+            </button>
+          </article>
+        )}
+      </section>
+      <article className="hometext-container">
+        <p className="hometext">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.jsx]: hand-cut phrase?

File: frontend/src/components/Carousel/Carousel.jsx [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+function Carousel() {
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  const handleSlideChange = (e) => {
+    const currentIndex = e.item;
+    setCurrentSlide(currentIndex);
+  };
+  return (
+    <>
+      <section className="carousel-container">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.jsx]: Wrong tag html

File: frontend/src/components/Carousel/Carousel.jsx [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+        <AliceCarousel
```
Comment (Yoann-Fortin) [inline frontend/src/components/Carousel/Carousel.jsx]: Keyboard navigation impossible with this library

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Romlion — APPROVED** [review]

**Review by pagouillart — APPROVED** [review]

**Review by Cyrildab — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**